apps/meteor/app/api/server/v1/oauthapps.ts

```diff
@@ -27,11 +27,12 @@ API.v1.addRoute(
 	{ authRequired: true, validateParams: isOauthAppsGetParams },
 	{
 		async get() {
-			if (!(await hasPermissionAsync(this.userId, 'manage-oauth-apps'))) {
-				return API.v1.unauthorized();
-			}
+			const isOAuthAppsManager = await hasPermissionAsync(this.userId, 'manage-oauth-apps');
 
-			const oauthApp = await OAuthApps.findOneAuthAppByIdOrClientId(this.queryParams);
+			const oauthApp = await OAuthApps.findOneAuthAppByIdOrClientId(
+				this.queryParams,
```

Comment:
I think it makes sense to sanitize 'em. If there's a chance of users being able to pass objects, we should definitely validate and sanitize them

Reply:
The params passed being a string should be validated by ajv on the `validateParams` key.

Would be nice to check on the schema to find if there's a way to sneak something other than an string there.

Reply:
> The params passed being a string should be validated by ajv on the `validateParams` key.

Exactly. Due to ajv we know exactly that these params are strings. I'll add some tests to validate this
I'm not sure about the regex validation, though. That's something we could have in ajv, but since there is no such validation on creation I'm afraid we could miss something